inc/views/nav_walker.php

```diff
@@ -55,7 +62,7 @@ public function inline_style_for_sidebar() {
 		if ( self::$add_mobile_caret_button_style ) {
 			return;
 		}
-		echo '<style>' . wp_kses_post( $this->get_mobile_submenu_style() ) . '</style>';
+		echo '<style>' . $this->get_mobile_submenu_and_accessibility_style() . '</style>'; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Would wp_kses_post alter the inline styles added here?

Reply:
Yes, it would break `>` direct descendent referencing, I could not keep that. But since we have control over the style content I think we can skip escaping that since we don't use any filters and we know the content.

Reply:
Makes sense, @preda-bogdan! 👍🏻 Just wanted to check 🚀